transformerlab/db.py

```diff
@@ -434,19 +434,20 @@ async def job_cancel_in_progress_jobs():
 ###############
 
 
-async def get_training_template(id):
+# async def get_training_template(id):
 
-    cursor = await db.execute("SELECT * FROM training_template WHERE id = ?", (id,))
-    row = await cursor.fetchone()
+#     cursor = await db.execute("SELECT * FROM training_template WHERE id = ?", (id,))
+#     row = await cursor.fetchone()
```

Comment:
I'm not sure I understand why you are commenting out this function?

Reply:
Oversight sorry, meant to comment out training_job_create